astropy/stats/tests/test_funcs.py

```diff
@@ -9,12 +9,19 @@
 from numpy.testing import assert_equal
 from numpy.testing.utils import assert_allclose
 
+from astropy.utils import minversion
+
 try:
     import scipy  # pylint: disable=W0611
 except ImportError:
     HAS_SCIPY = False
+    SCIPY_LT_1_4 = False
 else:
     HAS_SCIPY = True
+    if minversion(scipy, '0.14.0'):
```

Comment:
This could also be done in one line: ``SCIPY_LT_1_4 = minversion(scipy, '0.14.0')``.

On the other hand, why is this variable called like this? Shouldn't it be ``SCIPY_GT_0_14``?